### PR TITLE
Ruleset readme

### PR DESCRIPTION
**Pull request by jerzyn**
Fixes #

## Proposed Changes

Howto on spectral and rulesets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Refer to [RFC2119](https://www.ietf.org/rfc/rfc2119) for details.
 
 ### Questions & Comments
 
-_Please contact_ [_Zdenek.Nemec@externals.adidas-group.com_](mailto:Zdenek.Nemec@externals.adidas-group.com) _in the case of questions._
+_Please contact_ [_Zdenek.Nemec@externals.adidas-group.com_](mailto:Zdenek.Nemec@externals.adidas-group.com) or [_andrzej.jarzyna@adidas.com_](mailto:andrzej.jarzyna@adidas.com) _in the case of questions._
```
Comment (Amzani) [inline README.md]: @jerzyn add me as well please.

Comment (jerzyn) [reply README.md]: done

Comment (Amzani) [reply README.md]: @jerzyn we should also add some basic integration tests with real OAS files
1. positive test with a clean OAS file
2. example of failed test.

File: README.md [hunk 2]
```diff
@@ -41,9 +41,27 @@ The CAPITALIZED words throughout these guidelines have a special meaning:
 
 Refer to [RFC2119](https://www.ietf.org/rfc/rfc2119) for details.
 
+### Enforcing API Guidelines over your OpenAPI Specification
```
Comment (zdne) [inline README.md]: Enforcing -> is not a great word. I'd suggest something like "Validating your OpenAPI Specification with the adidas API guidelines"

File: README.md [hunk 3]
```diff
@@ -41,9 +41,27 @@ The CAPITALIZED words throughout these guidelines have a special meaning:
...
+
+In the `ruleset.md` file you can find a digest of enforceable rules over your API description documents. If you are using OpenAPI Specification as the API description format you can also leverage the `spectral.yaml` ruleset to automatically verify your specification compliance using [Spectral](github.com/stoplightio/spectral).
```
Comment (zdne) [inline README.md]: drop `enforcable`

**Pull request merged**